content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/start.rb

```diff
@@ -30,6 +32,21 @@ def service
                 end
               end
 
+              def playbook_ttl
+                service.options[:config_info][service_action.downcase.to_sym][:playbook_ttl].to_i
+              end
+
+              def retry_interval
+                ttl = playbook_ttl
+                return if ttl.zero? || @handle.root['ae_state_max_retries'].zero?
```

Comment:
Moving `@handle.root['ae_state_max_retries']` into a variable will make this method easier to read.

```ruby
def retry_interval
  ttl = playbook_ttl
  max_retry_count = @handle.root['ae_state_max_retries']

  return if ttl.zero? || max_retry_count.zero?
  ...
```